services/policies/pkg/engine/opa/rf_mimetype.go

```diff
@@ -23,7 +24,9 @@ var RFMimetypeExtensions = rego.Function1(
 		if err := ast.As(a.Value, &mt); err != nil {
 			return nil, err
 		}
-
+		if err := mime.AddExtensionType(".oform", "application/vnd.openxmlformats-officedocument.wordprocessingml.document"); err != nil {
```

Comment:
i prefer to rely on the system `mime.types` and extend those instead.

https://cs.opensource.google/go/go/+/master:src/mime/type_freebsd.go
https://cs.opensource.google/go/go/+/master:src/mime/type_dragonfly.go
https://cs.opensource.google/go/go/+/master:src/mime/type_openbsd.go
https://cs.opensource.google/go/go/+/master:src/mime/type_unix.go

...

we can add that function as a separate pr and rely on config values or even our own `.ocis/mimes.types`, what do you think?

Reply:
we should read from a config array and add our custom types

Reply:
instead of using a config slice i've decided to use RFC 4288 formatted files, ok for you?